topology.yml

```diff
@@ -0,0 +1,66 @@
+- server: 'edgenode2'
+  id: '5e9bdfc4-3a8f-47d2-b1ec-dc7ee3c11a85'
+  address: 192.168.16.22
+  port: 50051
+  device_capabilities:
+    model: 'NVIDIA RTX 4000 ADA GENERATION'
+    chip: 'NVIDIA'
+    memory: 20
+    fp32: 0
+    fp16: 0
+    int8: 0
+- server: 'edgenode3'
+  id: 'bbf0d2fe-a37c-46ce-be23-4a14f6c52c74'
+  address: 192.168.16.32
+  port: 50051
+  device_capabilities:
+    model: 'NVIDIA RTX 4000 ADA GENERATION'
+    chip: 'NVIDIA'
+    memory: 20
+    fp32: 0
+    fp16: 0
+    int8: 0
+- server: 'edgenode4'
+  id: 'cd44ef77-f5e3-4ad9-88b8-e64d3be5f66c'
+  address: 192.168.16.42
+  port: 50051
+  device_capabilities:
+    model: 'NVIDIA RTX 4000 ADA GENERATION'
+    chip: 'NVIDIA'
+    memory: 20
+    fp32: 0
+    fp16: 0
+    int8: 0
+- server: 'edgenode5'
+  id: '7fdeaeec-cd59-42ea-b1ac-a0fbbfeba2db'
+  address: 192.168.16.52
+  port: 50051
+  device_capabilities:
+    model: 'NVIDIA RTX 4000 ADA GENERATION'
+    chip: 'NVIDIA'
+    memory: 20
+    fp32: 0
+    fp16: 0
+    int8: 0
+- server: 'edgenode6'
+  id: 'a94ce36f-38d5-4542-be29-df20a6f17e40'
+  address: 192.168.16.62
+  port: 50051
+  device_capabilities:
+    model: 'NVIDIA RTX 4000 ADA GENERATION'
+    chip: 'NVIDIA'
+    memory: 20
+    fp32: 0
+    fp16: 0
+    int8: 0
+- server: 'edgenode7'
+  id: 'df11ebbf-8b51-4a55-86ab-eae3cfc7d6ca'
+  address: 192.168.16.72
+  port: 50051
+  device_capabilities:
+    model: 'NVIDIA RTX 4000 ADA GENERATION'
+    chip: 'NVIDIA'
+    memory: 20
+    fp32: 0
+    fp16: 0
+    int8: 0
```

Comment:
Can we put these into `flops`? So it would be

```
flops:
  fp32: 0
  fp16: 0
  int8: 0
```

Reply:
You sure, that add another loop if you want to read that ???

Reply:
Yes I'm sure it's cleaner for the user writing the configuration which is what matters.

Reply:
Done.